web/client/plugins/DrawerMenu.jsx

```diff
@@ -173,7 +175,8 @@ class DrawerMenu extends React.Component {
 
 const DrawerMenuPlugin = connect((state) => ({
     active: state.controls && state.controls.drawer && state.controls.drawer.active,
-    disabled: state.controls && state.controls.drawer && state.controls.drawer.disabled
+    disabled: state.controls && state.controls.drawer && state.controls.drawer.disabled,
+    isMobile: state.browser?.mobile
```

Comment:
Why we need to change the drawer menu? it's not a component used by the identify